src/components/accessWrappers/ProjectRoute.tsx

```diff
@@ -14,19 +15,29 @@ import { generateSafePath } from '@/utils/pathEncoding';
  */
 const ProjectRoute: React.FC<
   React.PropsWithChildren<{
-    permissions: ProjectPermissions | ProjectPermissions[];
+    permissions?: ProjectPermissions | ProjectPermissions[];
     redirectRoute?: string;
+    dataCollectionFeature?: DataCollectionFeatureRole;
   }>
-> = ({ permissions, redirectRoute, children }) => {
+> = ({ permissions, redirectRoute, dataCollectionFeature, children }) => {
   const { project } = useProjectDashboardContext();
   const [hasPermission, { loading }] = useHasProjectPermissions(
     project.id,
-    ensureArray(permissions)
+    permissions ? ensureArray(permissions) : []
```

Comment:
`ensureArray` should handle the case when permissions is null or undefined- that util is meant to help us avoid ternaries like this!

Reply:
Sounds good, `ensureArray` just needed a small typescript update to allow. https://github.com/greenriver/hmis-frontend/pull/959/files#diff-d03a2dd2397fbec94efa7e9b0cb974eada2c960a0b3c964fe6b4822c9e809fbbR1